.github/workflows/flink-remote-shuffle-tests.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Build and test with maven
-        run: mvn clean install -DskipTests; mvn -PincludeE2E -B clean verify
+        run: mvn -U clean install -DskipTests; mvn -PincludeE2E -B clean verify
```

Comment:
This is not necessary, I did this change because I was suspecting that the dependency is not update which cause the dependency not found exception. After add the apache snapshot repo, the issue is solved. Will remove this change.